pyazhpc/azhpc.py

```diff
@@ -530,23 +534,17 @@ def do_slurm_resume(args):
 
     log.info(f"slurm resume for {args.nodes}")
     # first get the resource name
-    resource_name, brackets = re.search(r'([^[]*)\[?([\d\-\,]*)\]?', args.nodes).groups(0)
-    resource_list = []
-    if bool(brackets):
-        for part in brackets.split(","):
-            if "-" in part:
-                lo, hi = part.split("-")
-                assert len(lo) == 4, "expecting number width of 4"
-                assert len(hi) == 4, "expecting number width of 4"
-                for i in range(int(lo), int(hi) + 1):
-                    resource_list.append(f"{resource_name}{i:04d}")
-            else:
-                assert len(part) == 4, "expecting number width of 4"
-                resource_list.append(f"{resource_name}{part}")
-    else:
-        resource_list.append(resource_name)
-        resource_name = resource_name[:-4]
-    
+    resource_name, resource_list = _nodelist_expand(args.nodes)
+
+    # Generate node/sku lookup table from nodes.conf
+    sku_lookup = {}
+    for line in open('/apps/slurm/nodes.conf', 'r'):
```

Comment:
we cannot force this location to be used

Reply:
@edwardsp I have kept the original file organization where all the Slurm configuration files are stored in `/apps/slurm` and then linked in `/etc/slurm`. A more robust approach would be to install the files directly in `/etc/slurm`.

Reply:
I think `/apps/slurm` is fine for just one example (as we could create another config with a different location) but this is putting it into the azurehpc code.